model/src/test/kotlin/config/OrtConfigurationTest.kt

```diff
@@ -21,6 +21,7 @@ package org.ossreviewtoolkit.model.config
 
 import io.kotest.assertions.throwables.shouldThrow
 import io.kotest.core.spec.style.WordSpec
+import io.kotest.extensions.system.withEnvironment
```

Comment:
Commit message *title*: Please use

    OrtConfigurationTest: Add test case for using environment variables

Reply:
s/ORTConfigurationTest/OrtConfigurationTest/ (really just use the literal file name)